pkgs/applications/office/gtg/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, python3Packages
+, pkgconfig
+, ninja
+, gtk3
+, wrapGAppsHook
+, glib
+, itstool
+, gettext
+, pango
+, gdk-pixbuf
+, gobject-introspection
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gtg";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+      owner = "getting-things-gnome";
+      repo = "gtg";
+      rev = "6623731f301c1b9c7b727e009f4a6462ad381c68";
+      sha256 = "14gxgg4nl0ki3dn913041jpyfhxsj90fkd55z6mmpyklhr8mwss1";
+  };
+
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    wrapGAppsHook
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    glib
+    gtk3
+    itstool
+    gettext
+    pango
+    gdk-pixbuf
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pycairo
+    pygobject3
+    lxml
+    dbus-python
+    gst-python
+    liblarch
+    pyxdg # can probably be removed after next release
+  ];
+
+  format = "other";
+  strictDeps = false;
+
+  meta = with stdenv.lib; {
+    description = "
+      Getting Things GNOME! (GTG) is a personal tasks and TODO-list items organizer for the GNOME desktop environment and inspired by the ''Getting Things Done'' (GTD) methodology.
```

Comment:
This is too long.

Reply:
```
    description = "
      Getting Things GNOME! (GTG) is a personal tasks and TODO-list items organizer inspired by the ''Getting Things Done'' (GTD) methodology.
    ";
```
Better or still too long?

Reply:
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

Reply:
Done.
```nix
    description = " A personal tasks and TODO-list items organizer.";
    longDescription = ''
      "Getting Things GNOME" (GTG) is a personal tasks and ToDo list organizer inspired by the "Getting Things Done" (GTD) methodology.
      GTG is intended to help you track everything you need to do and need to know, from small tasks to large projects.
    '';
```